### ISSUE TITLE
Frame by frame representation of moves

### ISSUE THREAD

**Issue by insanetesterftw** [cutechess/cutechess#171]
This proposal is perhaps too costly. But I'm hopeful. :)

![cutechessframebyframe_upload](https://cloud.githubusercontent.com/assets/11148841/19114674/d745819c-8b06-11e6-9260-42d6810f13d9.png)

The image attached shows what would look like.

The placement of the frames (boards) and their size are for illustrative purposes only.

Some options could be given:

- placement of the frames
- size of the frames
- number of frames
- each frame should include the arrows movements

**Comment by ilaripih** [comment]
Yes, maybe a bit too costly and and wild to render tons of chessboards. But since Cute Chess already has a movie player-like game viewer, we could add a mouseover tooltip to the slider showing the move number and a thumbnail of the board, similar to what Youtube does with the slider tooltip.
